core/contract/src/main/java/org/eclipse/dataspaceconnector/contract/common/ContractValidationException.java

```diff
@@ -0,0 +1,23 @@
+/*
+ *  Copyright (c) 2022 Fraunhofer Institute for Software and Systems Engineering
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Fraunhofer Institute for Software and Systems Engineering - initial API and implementation
+ *
+ */
+package org.eclipse.dataspaceconnector.contract.common;
+
+import org.eclipse.dataspaceconnector.spi.EdcException;
+
+public class ContractValidationException extends EdcException {
```

Comment:
This class seems very specific to contract validation as opposed to being a common exception type. Instead of moving it to that package, I think we may be able to get rid of it altogether (additional comments to follow).

Reply:
I will remove it.